pc/src/main.rs

```diff
@@ -97,23 +97,34 @@ async fn run_game() {
 	let mut game_state = GameState::new();
 	let mut events = game_state.input_players.get_events();
 	let mut no_player = true;
-	/*while no_player {
-		events = game_state.input_players.get_events();
-		for event in events.iter() {
-			if event.is_some(){
-				no_player = false;
-				println!("{:?}", event);
+
+	match &mut game_state.game_run {
+		None => todo!("no game run"),
+		Some(ref mut game_run) => {
+			while no_player {
+				events = game_state.input_players.get_events();
+				for event in events {
+					if event.is_some() {
+						no_player = false;
+						break;
+					}
+				}
+				next_frame().await;
+			}
+	
+			let players = Players::init();
```

Comment:
Player is already init [here](https://github.com/LuckyTurtleDev/m3/blob/f4e30c99e8647839b985dd62470c8d9a34b36ff9/pc/src/main.rs#L87). It can only init once.